### DISCUSSION TITLE
Why is my instance version marked with "+dirty" at the end?

### DISCUSSION THREAD

**Discussion by andersnordh** [searxng/searxng#3805] [Q&A] [answered]
Hi, as the title say, why?

I know that it has to do with me switching the front page image for my own but I make sure to set the same owner and file rights so ... ???

Regards,
Anders

**Comment by allendema** [comment, marked as answer]
https://github.com/searxng/searxng/blob/5cca3f6ef2a041fbf46d37abd52525dd700c4a85/searx/version.py#L67
